apps/address-service/domains/address/constants.js

```diff
@@ -8,9 +8,15 @@ const OVERRIDING_ROOT = 'meta.data'
 // ERRORS
 //
 const SOURCE_ALREADY_EXISTS_ERROR = 'SOURCE_ALREADY_EXISTS_ERROR'
+const EMPTY_SOURCE = 'EMPTY_SOURCE'
+const EMPTY_ADDRESS = 'EMPTY_ADDRESS'
+const INCORRECT_ADDRESS_SOURCE = 'INCORRECT_ADDRESS_SOURCE'
```

Comment:
It will be good to use a unified way for naming errors.
```suggestion
const EMPTY_SOURCE_ERROR = 'EMPTY_SOURCE_ERROR'
const EMPTY_ADDRESS_ERROR = 'EMPTY_ADDRESS_ERROR'
const INCORRECT_ADDRESS_SOURCE_ERROR = 'INCORRECT_ADDRESS_SOURCE_ERROR'
```